apps/agent-service/src/interface/agent-service.interface.ts

```diff
@@ -37,7 +37,6 @@ export interface IAgentConfigure {
   apiKey: string;
   orgId: string;
   network?: string;
-  tenant?: boolean;
```

Comment:
Please make sure this will not break tenancy flow

Reply:
Yes I tested and tenantId by default taking null value